src/utils/errorHelper.ts

```diff
@@ -0,0 +1,10 @@
+export function checkError(response: Response) {
+  if (!response.ok) {
+    throw Error(response.statusText);
+  }
+  return response;
+}
+
+export function handleError(error: Error) {
+  throw console.warn(error);
```

Comment:
Do we really need to throw `console.warn`?

Reply:
I'll move the error handling logic to concrete requests because if there is no error status/message  without a request context, it is hard to process it normally